### DISCUSSION TITLE
Big Sur Homebrew 2.5.11 httpd mariadb php@7.3

### DISCUSSION THREAD

**Discussion by EagleSH** [Homebrew/discussions#156] [Everyday usage] [answered]
Hello, guys can i upgrade from Catalina, this will work for my needs?

Homebrew 2.5.11 apache mariadb php@7.3 

MBP 16 Intel

**Comment by SMillerDev** [comment]
Well, Big Sur isn't officially supported by homebrew yet. It might work, it might not.

**Reply by EagleSH** [reply]
and there no eta when it will be officially supported, right? 

maybe someone already tried, any experience is appreciated

**Reply by SMillerDev** [reply]
We're a volunteer run project, we can't give ETAs on when people will use some of their spare time to help Homebrew.

**Comment by EagleSH** [comment, marked as answer]
maybe somebody will be helpful, just updated to Big Sur, everything works perfect
